File: PythonDataAnalysis/Renaming Headers and Variables.R
# add variables when import data
NH_dat = read.csv("Desktop/Github/PythonLearning/PythonDataAnalysis/Dataset_NoHead.csv")
colnames(NH_dat)=c('ID','Payment','Benefit_Amt')
head(NH_dat)

PH_dat = read.csv("Desktop/Github/PythonLearning/PythonDataAnalysis/Dataset_PartHead.csv")
names(PH_dat)[2]='Payment_Date'
# or
colnames(NH_dat)[2]='Payment_Date'
head(PH_dat)

File: PythonDataAnalysis/Renaming Headers and Variables.py
import pandas as pd

# name the header when import the data
Data_header = ['ID','Payment','Benefit_Amt']
NH_dat = pd.read_csv('Dataset_NoHead.csv', names=Data_header)
NH_dat.head()

# or 
NH_dat1 = pd.read_csv('Dataset_NoHead.csv', header = None)
NH_dat1.columns = Data_header

# Data with partial headers and renaming variables
PH_dat = pd.read_csv('Dataset_PartHead.csv')
PH_dat.head()
PH_dat = PH_dat.rename(columns={'Unnamed: 1':'Payment_Date'})